lib/errors/warning_test.go

```diff
@@ -0,0 +1,23 @@
+package errors
+
+import (
+	"testing"
+
+	"github.com/cockroachdb/errors"
```

Comment:
```suggestion
	"github.com/sourcegraph/sourcegraph/lib/errors"
```

This package is used now instead of a cockroach one (which in fact is also a wrapper of a cockroach errors)

Reply:
Good catch. I fixed this in a commit as this suggestion wouldn't have worked and being in the same package already we can directly call `As()`.